lib/modules/rest/index.js

```diff
@@ -331,8 +331,9 @@ Rest.prototype.handleRequest = function(req, res, $happn, $origin) {
               return;
             }
 
-            if (body.parameters[parameter.name]) args.push(body.parameters[parameter.name]);
-            else args.push(null);
+            if (body.parameters[parameter.name] || body.parameters[parameter.name] === 0) {
```

Comment:
Should we also check for orher nullish values here - empty string, false etc

Reply:
Looking at fieldpop-api, setDeviceData could want to set a value that could unexpectedly fail here... Not exactly sure what we should cater for? Otherwise we could do a (body.parameters[parameter.name] !== undefined)

Reply:
I like the check for not undefined the most

Reply:
updated